phone-scripts/02-install-packages.sh

```diff
@@ -18,6 +18,9 @@ infecho "This adds my COPR repository (njha/mobile) and installs phone related p
 infecho "Only functional on Fedora Rawhide."
 infecho "HEAVY WIP, untested"
 
+infecho "Link temp-resolv.conf"
+ln -sfv /etc/tmp-resolv.conf /etc/resolv.conf
```

Comment:
Link to copied host resolv.conf from inside chroot (this actually worked, linking from outside caused errors)